community/kernel/src/main/java/org/neo4j/kernel/impl/transaction/log/entry/LogEntryWriterV1.java

```diff
@@ -86,11 +86,10 @@ public void serialize( TransactionRepresentation tx ) throws IOException
     }
 
     @Override
-    public void writeCheckPointEntry( long transactionId, LogPosition logPosition ) throws IOException
+    public void writeCheckPointEntry( LogPosition logPosition ) throws IOException
     {
         writeLogEntryHeader( CHECK_POINT );
-        channel.putLong( transactionId ).
-                putLong( logPosition.getLogVersion() ).
+        channel.putLong( logPosition.getLogVersion() ).
```

Comment:
Btw why do we store the log position in the checkpoint record? Just transaction id should be enough.